docs/en/administration/query_profile_details.md

```diff
@@ -0,0 +1,1366 @@
+
+##### QueryPeakMemoryUsage
+
+Description: Maximum peak memory across all compute nodes.
+
+##### QueryExecutionWallTime
+
+Description: Wall time of the execution.
+
+##### QueryCumulativeCpuTime
+
+Description: Cumulative CPU time across all compute nodes.
+
+##### QueryCumulativeOperatorTime
+
+Description: Cumulative time across all nodes. This is a simple linear accumulation, but in reality, execution times of different operators may overlap. This parameter serves as the denominator for calculating the percentage of time spent on each operator.
+
+##### QueryCumulativeNetworkTime
+
+Description: Cumulative network time of all Exchange nodes. Similar to cumulative operator time, actual execution times of different Exchanges may overlap.
+
+##### QueryCumulativeScanTime
+
+Description: Cumulative IO time of all Scan nodes. Similar to cumulative operator time, actual execution times of different Scan operations may overlap.
+
+##### QueryPeakScheduleTime
+
+Description: Maximum ScheduleTime metric across all Pipelines.
+
+##### QuerySpillBytes
+
+Description: Number of spilled bytes.
+
+##### ResultDeliverTime
+
+Description: Additional time to transfer results. For query statements, this parameter refers to the time it takes to send data back to the client; for insert statements, it refers to the time it takes to write data to the storage layer.
+
+### Fragment Metrics
+
+##### InstanceNum
+
+Description: Number of all FragmentInstances for this Fragment.
+
+##### InstanceIds
+
+Description: IDs of all FragmentInstances for this Fragment.
+
+##### BackendNum
+
+Description: Number of BEs participating in the execution of this Fragment.
+
+##### BackendAddresses
+
+Description: Addresses of all BEs participating in the execution of this Fragment.
+
+##### FragmentInstancePrepareTime
+
+Description: Time spent in the Fragment Prepare phase.
+
+##### InstanceAllocatedMemoryUsage
+
+Description: Cumulative allocated memory for all FragmentInstances under this Fragment.
+
+##### InstanceDeallocatedMemoryUsage
+
+Description: Cumulative deallocated memory for all FragmentInstances under this Fragment.
+
+### Pipeline Metrics
+
+The relationship between core metrics is illustrated in the following diagram:
+
+- DriverTotalTime = ActiveTime + PendingTime + ScheduleTime
+- ActiveTime = ∑ OperatorTotalTime + OverheadTime
+- PendingTime = InputEmptyTime + OutputFullTime + PreconditionBlockTime + PendingFinishTime
+- InputEmptyTime = FirstInputEmptyTime + FollowupInputEmptyTime
+
+![profile_pipeline_time_relationship](../assets/Profile/profile_pipeline_time_relationship.jpeg)
+
+##### DegreeOfParallelism
+
+Description: Degree of parallelism.
+
+##### TotalDegreeOfParallelism
+
+Description: Sum of degrees of parallelism. Since the same Pipeline may execute on multiple machines, this aggregates all parallelism.
+
+##### DriverPrepareTime
+
+Description: Time to execute Prepare, excluding it from DriverTotalTime.
```

Comment:
Time to execute Prepare. This metric is  excluded from DriverTotalTime.
Comment